src/peakWatcher.cpp

```diff
@@ -397,8 +404,9 @@ void CALLBACK tick(HWND hwnd, UINT msg, UINT_PTR event, DWORD time) {
 	ostringstream s;
 	s << fixed << setprecision(1);
 	const bool multiple = isWatchingMultipleValues();
+	const ReaProject* proj = currentProject();
 	for (int w = 0; w < NUM_WATCHERS; ++w) {
-		Watcher& watcher = watchers[w];
+		Watcher& watcher = watchers[proj][w];
```

Comment:
This isn't a big issue since we only have 2 watchers, but this means we're doing a map lookup for project twice when we don't need to. Perhaps we could instead get the array of watchers for the project:

```
const auto& projWatchers = watchers[currentProject()];
for (int w = 0; w < NUM_WATCHERS; ++w) {
	Watcher& watcher = projWatchers[w];
```